server/src/main/java/org/cloudfoundry/identity/uaa/zone/MultitenantJdbcClientDetailsService.java

```diff
@@ -12,11 +12,22 @@
  *******************************************************************************/
 package org.cloudfoundry.identity.uaa.zone;
 
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import javax.sql.DataSource;
+
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
-import org.cloudfoundry.identity.uaa.audit.event.SystemDeletable;
```

Comment:
These, and others, were probably removed by Eclipse because they were unused imports and got cleaned up when I told it to clean out the various interfaces that were consolidated into `MultitenantClientDetailsService`.